pkgs/ts/@circles-pink/web-client/src/onboarding/utils/paginate.ts

```diff
@@ -0,0 +1,132 @@
+export const paginate = (
+  totalItems: number,
+  currentPage = 1,
+  pageSize = 5,
+  maxPages = 10000
+) => {
+  // calculate total pages
+  let totalPages = Math.ceil(totalItems / pageSize);
+
+  // ensure current page isn't out of range
```

Comment:
I'd suggest not to override the argument, instead introduce a new `const` variable and assign the value with an expression (see comment above).